src/noisepy/seis/pnwstore.py

```diff
@@ -110,7 +123,7 @@ def read_data(self, timespan: DateTimeRange, chan: Channel) -> ChannelData:
 
         # reconstruct the file name from the channel parameters
         chan_str = f"{chan.station.name}.{chan.station.network}.{timespan.start_datetime.strftime('%Y.%j')}"
-        filename = fs_join(self.paths[timespan.start_datetime], f"{chan_str}")
+        filename = fs_join(self.paths[timespan.start_datetime].replace("__", chan.station.network), f"{chan_str}")
```

Comment:
I'd suggest something more explicit than `__`, e.g. `{NETWORK}`

Reply:
So this is a tricky one. "__" is actually the wildcard grammar of sqlite shown in the filename path. In the Jupyter notebook, the data path is actually defined as (see below) to load all network.
```
DATA = "/1-fnp/pnwstore1/p-wd00/PNW2020/__/"  
```

The easiest way to allow read_data() work is to simply replace "__" with the network code.

Reply:
Gotcha, makes sense!